<launch>

        <node name="road_information" pkg="pharos_road_information" type="road_information" required="true" output="screen">
		<param name="road_data_file" value="$(find pharos_road_information)/waypoints/KAIST/merged_wp.txt" />
		<param name="reverse_waypoint" value="false" type="bool"/>
        </node> 

    <!-- <include file="$(find pharos_trajectory_observer)/launch/trajectory_observer.launch" /> -->
    
</launch>
